File: chapter5/mario/mario.py
from cs50 import get_int

n = 0

while (n < 1) or (n > 8):
    n = get_int("Height: ")
        
cnt = n    
x = n
        
for r in range(n):
    cnt = n 
    for c in range(n):
        if (cnt > r + 1):
            print(" ",end = "")
        else: 
            print("#",end = "")
        cnt -= 1    
            
    print("  ",end = "")
    
    
    for a in range(r+1):
        print("#",end = "")
    print("")

File: chapter1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main (void) {
    int n = 0;
    
    do {
        
        n = get_int("Height: ");
        
    } while (n < 1 || n > 8);
    
    
    for (int r = 0; r < n; r++){
        
        for(int c = n; c > 0; c--){
            
            if (c > r + 1){
                printf(" ");
            } else {
                printf("#");
            }
        }
        
        printf("  ");
        
        for(int a = 0; a < (r+1); a++){
            
                printf("#");
            
        }
        
      printf("\n");
        
    }
    
}